src/reporting/market.se

```diff
@@ -140,16 +144,16 @@ def disputeAutomatedReport():
     # intentionally does not migrate the market as automated report markets won't actually migrate unless a dispute bond has been placed or the automated report doesn't occur
     require(not self.isFinalized())
     require(self.isInAutomatedDisputePhase())
-    self.getReputationToken().transferFrom(msg.sender, self, AUTOMATED_REPORTER_DISPUTE_BOND_AMOUNT)
-    self.automatedReporterDisputeBondHolder = msg.sender
+    if (not self.automatedReporterDisputeBondToken):
```

Comment:
This if should instead be `require(not self.automatedReporterDisputeBondToken)`.  None of the code in this function should execute if there is already a disputer.

The same goes for the other two similar functions.